content/20.treatments.md

```diff
@@ -226,6 +226,16 @@ In principle, drugs that reduce the expression of these proteins or sterically h
 
 A list of current studies and their results, using carefully the information requested in the therapeutic paper tickets.
 
+Angiotensin-converting enzyme inhibitors and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications, prescribed well over 100,000,000 times annually in the United States, for example.[@url:https://clincalc.com/DrugStats/Drugs/Lisinopril; @doi:10.1056/NEJMp1901657]
+Data from some animal models suggest several but not all ACE inhibitors and several ARBs increase ACE2 expression in cells of some organs.[@doi:10.1093/cvr/cvaa097]
+Clinical studies have not established that plasma ACE2 expression is increased in humans treated with these medications.[@doi:10.1093/europace/euw246]
+While randomized clinical trials are ongoing, a variety of observational studies have examined the relationship between exposure to ACE inhibitors or ARBs and outcomes in patients with COVID-19.
+Mehra and colleagues reported a 0.33 odds of death in hospitalized patients with COVID-19 who were treated with an ACE inhibitor at time of admission.[@doi:10.1056/NEJMoa2007621]
```

Comment:
```suggestion
Mehra and colleagues reported a 0.33 odds of death in hospitalized patients with COVID-19 who were treated with an ACE inhibitor at time of admission [@doi:10.1056/NEJMoa2007621].
```

Reply:
The relevant sentence in their abstract is: "No increased risk of in-hospital death was found to be associated with the use of ACE inhibitors (2.1% vs. 6.1%; odds ratio, 0.33; 95% CI, 0.20 to 0.54) or the use of ARBs (6.8% vs. 5.7%; odds ratio, 1.23; 95% CI, 0.87 to 1.74)."  So I think that the 0.33 was *not* an odds, but an odds ratio (ratio of odds between conditions).  An odds of 0.33 would mean that the death rate amoung ACE users was 25% (i.e., 1:3 odds), whereas it was actually 2%.  The ACE users have a significantly lower probability of death than non-ACE users, but the authors don't actually report this as a finding, apparently because they are only reporting observational risk factors and not trying to use observational data to advocate for a treatment.

Reply:
Thank you. I fully agree about it being an odds ratio. They are reporting it as a finding, however, as far as I can tell. I'm not sure I understand the distinction you're drawing between a finding and something else reported. If they leave the reader with the impression that treatment with a medication is a risk factor (as opposed to a risk marker), they're speaking of the effect of the drug, albeit indirectly.

Reply:
Thanks @dziakj1 ! I knew something felt off, this really helps. Above is modified per your comment!

Reply:
Sorry, I wasn't clear.  What I meant was that they said that there was "no increased risk," rather than saying that there was "decreased risk," as they could have because the difference was in fact statistically significant for a two-sided test.

Reply:
This threw me too. It's actually a logistic regression, and I'm not sure what to make of that with respect to one-tailed vs two-tailed!

Reply:
The accompanying editorial (doi:@10.1056/NEJMe2012924) focuses on the ARB / ACE inhibitor findings, which are the findings that could change practice if interpreted as having a cause-effect relationship with outcomes.

Reply:
Thanks, @rando2!  I think that when we report an odds ratio, we should tell both conditions being compared.  An odds ratio is always saying that there was less risk in one condition than in another, so it doesn't make sense unless we tell the conditions.  2% of the users and 6% of the non-users died in hospital.  It seems like they could have used this to argue that ACE inhibitors are protective, but didn't want to, because of the observational nature of the data.

Reply:
@dziakj1 for some reason, I'm struggling with the grammar here! Hopefully this suggestion gets a bit closer to correct!

Reply:
Yes, I like the new version much better: .33 odds ratio relative to other patients. Thanks @rando2  and @byrdjb!

Reply:
Good discussion! Thank you, @dziakj1 & @rando2 for very helpful corrections and suggestions.

Reply:
It probably wouldn't even hurt to add "2.1% vs. 6.1%" to avoid misunderstandings, but that is up to you.

Reply:
@rando2   I think that maybe they were thinking of their test as one-tailed, even though the software did a two-tailed test.  That is, they decided not to report the finding because they weren't looking for it.  This is an extremely cautious, conservative approach, but isn't wrong.